lascli/parser/transitions.py

```diff
@@ -98,6 +110,10 @@ def create_transitions_parser(subparsers):
     execute_parser.add_argument('transition_id')
     execute_parser.set_defaults(cmd=execute_transition)
 
+    delete_parser = subparsers.add_parser('delete')
+    delete_parser.add_argument('transition_id')
+    delete_parser.set_defaults(cmd=delete_transition)
```

Comment:
Should we add a small comment: "Will fail if transition is in use by one or more workflows"?